config/locales/en/shared.yml

```diff
@@ -2,5 +2,5 @@ en:
   shared:
```

Comment:
Not sure we will be reusing this in other places. Might be more clean to keep it all together for now in the `dashboard.yml` index section? Perhaps `crime_applications.index.status.{xxx}`
We have also in there the table headings so kind of flows nice.

Reply:
Makes sense I will amend 👍